lib/cocoapods/installer/xcode/pods_project_generator/aggregate_target_installer.rb

```diff
@@ -35,6 +35,7 @@ def install!
               create_copy_resources_script
               create_acknowledgements
               create_dummy_source(native_target)
+              clean_support_files_temp_dir
```

Comment:
This was previously being called every time a generated file was added to the support files group. Changed it to be called once at the end of the installation